website/docs/r/integer.html.md

```diff
@@ -23,7 +23,7 @@ a `aws_alb_listener_rule` resource:
 ```hcl
 resource "random_integer" "priority" {
   min     = 1
-  max     = 99999
+  max     = 50000
```

Comment:
Can you change the line above this to match the new example? 

> The following example shows how to generate a random priority between 1 and 50000 for
a `aws_alb_listener_rule` resource: